sdv/datasets/local.py

```diff
@@ -1,7 +1,9 @@
 """Methods to load local datasets."""
 
 import warnings
-from os import path, walk
+from os import makedirs, path, walk
```

Comment:
Can we import `os` and use `os.path` `os.makedirs` etc.. it is a bit more explicit. Also `path` makes me think of `pathlib` and it  can be a bit confusing.

Reply:
Do we have a preference between `os.path` and `pathlib` for situations like this? I usually end up using `pathlib`, but not sure if there's a standard.

Reply:
`pathlib` is the way to go when it comes to `paths`, eg: exploring paths, joining paths etc.. but when it comes to interact with the system (like creating a folder etc..) since `pathlib` doesn't offer that functionality `os` is the library of preference.